src/ui/components/CardsStack/CardsStack.tsx

```diff
@@ -56,12 +57,18 @@ const CardsStack = ({
     );
   };
 
-  const handleShowCardDetails = (index: number) => {
+  const handleShowCardDetails = async (index: number) => {
     setIsActive(true);
     let pathname = "";
 
     if (cardsType === CardType.IDENTIFIERS) {
       const data = cardsData[index] as DIDDetails;
+      /**The below code only return false if the identifier is a multisig and it is not ready */
+      const checkMultisigComplete =
+        await AriesAgent.agent.identifiers.checkMultisigComplete(data.id);
+      if (!checkMultisigComplete) {
+        return;
+      }
```

Comment:
@sdisalvo-crd Is this OK? If you click on a multi-sig that is pending now - if it's after completing in the background, the identifier will go from pending to complete and it will show the identifier details.

If it's still not ready, nothing will happen.

(Bearing in mind that when Leo has notifications for this, we can remove this logic and it will simply move from pending to complete automatically without this "click")

Reply:
@Sotatek-HocNguyena Since we have a new design on where to place pending identifiers this will change. Can you revert changes to the `ui` directory? Sal will add this in the right place in his PR after.

Reply:
@Sotatek-HocNguyena nevermind actually lets keep it so Sal can easily see what to move in his PR after so disregard this comment

Reply:
@iFergal Thanks, go ahead as agreed on chat 👍

Reply:
Ok, I will keep it.